dev/README_RELEASE_AIRFLOW.md

```diff
@@ -246,6 +247,27 @@ Production Docker images should be manually prepared and pushed by the release m
 This will wipe Breeze cache and docker-context-files in order to make sure the build is "clean". It
 also performs image verification before pushing the images.
 
+## Prepeare issue for testing status of rc
```

Comment:
```suggestion
## Prepeare issue for testing status of rc

For now this part works for bugfix releases only, for major releases we will experiment in the future and
see if there is a way to only extract those bugfixes and high-level changes to make the list manageable.

```

Reply:
You mean "minor" or "feature" -- Major is 3.0.0

Reply:
true